lib/matplotlib/axes/_axes.py

```diff
@@ -2434,7 +2434,7 @@ def bar(self, x, height, width=0.8, bottom=None, *, align="center",
         # checking and processing will be left to the errorbar method.
         xerr = kwargs.pop('xerr', None)
         yerr = kwargs.pop('yerr', None)
-        error_kw = kwargs.pop('error_kw', {})
+        error_kw = kwargs.pop('error_kw', {}).copy()
```

Comment:
in anticipation that we might want to move from popping to kwargs, I'd standardize here
```suggestion
        error_kw = kwargs.pop('error_kw', None)
```
and before the first usage of `error_kw` below do
```
error_kw = {} if error_kw is None else error_kw.copy()  
```